### DISCUSSION TITLE
Attempt to read property "type" on null ( $bot->chat()->type )

### DISCUSSION THREAD

**Discussion by ghost** [nutgram/nutgram#276] [Q&A]
**Why am I getting this error?**

```[23-Dec-2022 12:11:54 Asia/Tehran] PHP Warning:  Attempt to read property "type" on null in /home/digisave/public_html/ds_beta/home/bc/app/Middlewares/CheckChatTypeMiddleware.php on line 11```

**My code:**
![image](https://user-images.githubusercontent.com/78247242/209304316-2fc105ad-1aff-4056-b4f5-649d0526e6ba.png)

How can i fix it ?

**Comment by Lukasss93** [comment]
Not all types of updates have the `Chat` object:
- inline_query
- chosen_inline_result
- shipping_query
- pre_checkout_query
- poll
- poll_answer

https://core.telegram.org/bots/api#update
